build/bootstrap.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
```

Comment:
Part of the motivation behind this script change is to be able to invoke it as an image factory artifact